src/Compilers/Server/VBCSCompilerTests/DesktopBuildClientTests.cs

```diff
@@ -70,16 +70,17 @@ public bool TryConnectToNamedPipeWithSpinWait(int timeoutMs, CancellationToken c
         public sealed class ServerTests : DesktopBuildClientTests
         {
             private readonly string _pipeName = Guid.NewGuid().ToString("N");
-            private readonly TempDirectory _tempDirectory;
             private readonly BuildPaths _buildPaths;
             private readonly List<ServerData> _serverDataList = new List<ServerData>();
             private bool _allowServer = true;
             private int _failedCreatedServerCount = 0;
 
             public ServerTests()
             {
-                _tempDirectory = Temp.CreateDirectory();
-                _buildPaths = ServerUtil.CreateBuildPaths(workingDir: _tempDirectory.Path);
+                var wo = Temp.CreateDirectory();
```

Comment:
It's not.  It's a line left over from an earlier version of the change.  Will delete.